src/extensions/template_extension/TemplateFactory.cs

```diff
@@ -15,6 +15,7 @@
 using Azure.AI.Details.Common.CLI;
 using Azure.AI.Details.Common.CLI.ConsoleGui;
 using System.Text.Json;
+using Newtonsoft.Json.Linq;
```

Comment:
Let's use `System.Text.Json` instead of `Newtonsoft`...

Reply:
fixed